### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by rmwoods**
This work represents an attempt to clean up our repository into a more manageable state going forward. It roughly follows the organization of [Cookie Cutter Data Science](https://drivendata.github.io/cookiecutter-data-science/), with some points taken from [this post](https://gist.github.com/ericmjl/27e50331f24db3e8f957d1fe7bbbe510) as well.

There is still work to do on this later on, but this at least sets up the directory structure.

* Finish `get_data.py` that pulls data from dropbox
* Create a `Makefile` that can reproduce data at different stages
* Create some data loading functions so we don't always have the same HDF loading code at the beginning of each notebook
* Probably some other things I'm forgetting...

I'm not quite sure why there are 230 commits, with a whole bunch from last year. This was branched from the most recent master...

### CODE REVIEW

File: beerai/data/vocabulary.py [hunk 1]
```diff
@@ -36,7 +44,11 @@ def _setup_argparser():
         description='Program to create a "vocabulatory" from a list of '
```
Comment (rlwelch) [inline beerai/data/vocabulary.py:44]: vocabulatory 🤓

**Review by rlwelch — APPROVED** [review]
Looks great! Happy to see the constants and frequently used filepaths moved to their own file.
Thanks for taking this on.

File: beerai/data/vocabulary.py (hunk 1 above)
Comment (rmwoods) [reply beerai/data/vocabulary.py:44]: XD

Comment (rmwoods) [reply beerai/data/vocabulary.py:44]: It's in quotes! Intentional!

**Pull request merged**